AGPs/AGP-W.md

```diff
@@ -0,0 +1,84 @@
+---
+AGP: N/A
+Title: Decentralizing `aragonpm.eth`
+Author: Jorge Izquierdo (@izqui)
+Status: Stage III
+Track: Association
+Created: 2019-04-03
+---
+
+## Motivation for changing this Association policy
+
+When Aragon One planned the Aragon 0.6 release, it was internally thought of as a soft-launch. Even though it was the mainnet deployment, the main aragonPM registry (`aragonpm.eth`) in both the rinkeby and mainnet environments was controlled by a cold key that Aragon One held. This has given us the ability to perform releases at any time, and act quick in case an issue were to be found.
+
+It's been more than 5 months after releasing Aragon 0.6. During this time we have published 4 minor releases, all of which were properly scheduled and no emergency releases have been done. We consider that it is now safe and appropriate to transition control of the aragonPM registries from the Aragon One cold key to a more decentralized and future-proof publishing process that won't solely depend on Aragon One.
+
+## Description of desired Association policy change
+
+The following changes are proposed to the `aragonpm.eth` aragonPM DAOs:
+
+- Mainnet and rinkeby environments:
+    - Change the root authority of the aragonPM DAOs from the 'Aragon One cold wallet' to the Aragon Association multisig. A Voting app within the DAO will control creating new repos and publishing new versions to all the repos (see Appendix for the proposed DAO structure)
+    - Create a new aragonPM registry at `a1.aragonpm.eth` controlled by Aragon One, granting 'Aragon One cold wallet' the `CREATE_REPO_ROLE` permission in the registry and setting this address as the root authority of the DAO. This registry can be used by Aragon One to publish apps before releasing them in `aragonpm.eth`.
```

Comment:
One potentially painful spot here is on how the CLI works with app names. Right now there's no easy way to modify the registry you want, without completely changing the app's name each time in its arapp.json.

Reply:
Could you please expand on this? Isn't this as easy as adding a new environment? 🤔

Reply:
Ah ha, yes, you're right! Forgot that the environments let you specify the full app name :).